Integrations/integration-IsItPhishing.yml

```diff
@@ -0,0 +1,166 @@
+description: Collaborative web service that provides validation on whether a URL is
+  a phishing page or not by analyzing the content of the webpage
+configuration:
+- display: Customer's name
+  name: name
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Customer's License
+  name: license
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Server URL
+  name: url
+  defaultvalue: https://ws.isitphishing.org
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |+
+    var auth = 'Bearer ' + Base64.encode(params.name + ':' + params.license);
```

Comment:
please use btoa (or atob, nut sure which way) instead of this base 64 encode which is very slow